### PR TITLE
CoPla - Companion Planting Ontology

### PR DESCRIPTION
**Pull request by pernisch**
Ontology which was developed as part of the project "Decision Support for Companion Planting" (https://github.com/kai-vu/companion-planting-decision-support)

### CODE REVIEW

File: CoPla/.htaccess [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Rewrite rule to serve HTML content
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^ontology\$ https://raw.githubusercontent.com/kai-vu/companion-planting-decision-support/refs/heads/main/owl/companion_planting_v6.owl [R=303,L]
+
+# Rewrite rule to serve RDF/XML content if requested
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^ontology\$ ontology.owl [R=303]
```
Comment (dgarijo) [inline CoPla/.htaccess:15]: you need to redirect to a URL. `ontology.owl` is not available here. Like you did in the redirection above.

File: CoPla/.htaccess [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+# Choose the default response
+RewriteRule ^ontology\$ ontology.owl [R=303]
```
Comment (dgarijo) [inline CoPla/.htaccess:18]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]
Please check the redirections

File: CoPla/.htaccess (hunk 1 above)
Comment (pernisch) [reply CoPla/.htaccess:15]: Thank you for the pointer! I'm fixing also the documentation right now, so will update this as well.

**Comment by pernisch** [comment]
I'm having trouble figuring out how to do this properly, sorry for the wait.
I generated the documentation using WIDOCO, so I have all the files but unsure of where to put them so all this works properly. WIDOCO also provides a much more complex than the one I've put here. If you have a pointer to a tutorial please send it my way :sweat_smile:

File: CoPla/.htaccess [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+AddType application/ld+json .jsonld
+# Rewrite engine setup
+RewriteEngine On
+#Change the path to the folder here
+RewriteBase https://github.com/kai-vu/companion-planting-decision-support/tree/main/owl/Companion%20Planting%20Ontology
```
Comment (davidlehn) [inline CoPla/.htaccess:14]: I'm pretty sure this feature is for local path rewrites and doesn't work with arbitrary URLs.  Did you test these rules?

**Comment by davidlehn** [comment]
Please add the github ids of maintainers to the README.

File: CoPla/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+- Which plant combinations should be avoided in a garden or farm setting?
+- What are the best companion plants for a specific crop (e.g., tomatoes)?
+- How can companion planting be used to combat specific pests (e.g., aphids)?
+
+Feel free to explore the ontology and contribute to its development. Your insights and feedback are valuable in enhancing this knowledge base.
+
+## Contact
+The mainteners of the ontology are researchers at the Knowledge in Artificial Intelligence group (kai.cs.vu.nl) at the Computer Science Department, Vrije Universiteit Amsterdam.
+
+**Currently reponsible persion**: Romana Pernisch (github: pernisch, email: r.pernisch@vu.nl)
```
Comment (dgarijo) [inline CoPla/README.md:27]:
```suggestion
**Currently reponsible person**: Romana Pernisch (github: pernisch, email: r.pernisch@vu.nl)
```

**Review by dgarijo — APPROVED** [review]

**Pull request merged**